### PR TITLE
Add snippet showing how to revert a modified file back to its previou…

### PR DESCRIPTION
**Pull request by JordanMartinez**
…s state in most recent commit

### CODE REVIEW

File: src/main/java/org/dstadler/jgit/porcelain/RevertChanges.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+                // add the file and commit it
+                git.add().addFilepattern(fileName).call();
+                git.commit().setMessage("Added untracked file " + fileName + "to repo").call();
+
+                // modify the file
+                Files.write(tempFilePath, "Some modifications".getBytes(), StandardOpenOption.APPEND);
+
+                // assert that file's text does not equal initialText
+                assert !initialText.equals(readFile(tempFilePath));
```
Comment (centic9) [inline src/main/java/org/dstadler/jgit/porcelain/RevertChanges.java]: Please don't use assertions here, these are likely turned off when this is run. Rather use normal error handling, e.g. 

```
if(!...) { 
    throw new IllegalStateException("Text should be different now") 
}
```

**Comment by centic9** [comment]
Thanks for the suggestion, can you also add a line in the README to list the new snippet there as well?

File: src/main/java/org/dstadler/jgit/porcelain/RevertChanges.java (hunk 1 above)
Comment (JordanMartinez) [inline src/main/java/org/dstadler/jgit/porcelain/RevertChanges.java]: K, I'll fix that.

**Comment by JordanMartinez** [comment]
> can you also add a line in the README to list the new snippet there as well?

Ah! Haha... Yeah that might help :stuck_out_tongue_winking_eye:

**Comment by centic9** [comment]
Thanks a lot, merged now!

**Pull request merged**